.travis.yml

```diff
@@ -1,12 +1,4 @@
 sudo: false
 language: node_js
 node_js:
-    - "0.10"
-    - "0.11"
-    - "0.12"
-    - "iojs"
-matrix:
-    allow_failures:
-        - node_js: "0.11"
-        - node_js: "0.12"
-        - node_js: "iojs"
+    - "4"
```

Comment:
one way could be use a script that detects the version and calls npm install {correct version}: https://docs.travis-ci.com/user/customizing-the-build/#Implementing-Complex-Build-Steps